src/cpp/rtps/builtin/discovery/endpoint/EDPSimple.cpp

```diff
@@ -249,10 +239,8 @@ bool EDPSimple::createSEDPEndpoints()
         watt.endpoint.multicastLocatorList = this->mp_PDP->getLocalParticipantProxyData()->m_metatrafficMulticastLocatorList;
         //watt.endpoint.remoteLocatorList = m_discovery.initialPeersList;
         watt.endpoint.durabilityKind = TRANSIENT_LOCAL;
-        watt.times.nackResponseDelay.seconds = 0;
-        watt.times.nackResponseDelay.fraction = 0;
-        watt.times.initialHeartbeatDelay.seconds = 0;
-        watt.times.initialHeartbeatDelay.fraction = 0;
+        watt.times.heartbeatPeriod.seconds = 1;
+        watt.times.heartbeatPeriod.fraction =0;
```

Comment:
Maybe use specific values for all the watt.times fields? Even better, have a const for it at the top of the file?

Reply:
Done